components/src/core/Drawer/DrawerFooter.tsx

```diff
@@ -1,27 +1,36 @@
 import React from 'react';
 import { makeStyles } from '@material-ui/core/styles';
 
+export type DrawerFooterProps = React.HTMLAttributes<HTMLDivElement> & {
+    backgroundColor?: string;
+    drawerOpen?: boolean;
+};
+
 const useStyles = makeStyles({
     root: {
         width: '100%',
+        backgroundColor: (props: DrawerFooterProps): string => props.backgroundColor,
     },
 });
 
-export type DrawerFooterProps = {
-    backgroundColor?: string;
-    drawerOpen?: boolean;
-};
-
 export const DrawerFooter: React.FC<DrawerFooterProps> = (props) => {
     const classes = useStyles(props);
-    const { children, backgroundColor, drawerOpen } = props;
+    const {
+        children,
+        drawerOpen,
+        // leaving those here to allow prop transferring
+        /* eslint-disable @typescript-eslint/no-unused-vars */
+        backgroundColor,
+        /* eslint-disable @typescript-eslint/no-unused-vars */
```

Comment:
I'll stop duplicating my comment now, but do we need these eslint/ts ignore lines if we are using the final spread variable later?

Reply:
We do need them, unfortunately.